src/io.h

```diff
@@ -78,6 +78,47 @@ struct TypInputFile {
 };
 
 
+/****************************************************************************
+**
+*/
+enum {
+    // the maximal number of used line break hints
+    MAXHINTS = 100,
+
+    // the widest allowed screen width
+    MAXLENOUTPUTLINE = 4096,
+};
+
+
+/****************************************************************************
+**
+*T  TypOutputFile . . . . . . . . . . structure of an open output file, local
+**
+**  'TypOutputFile' describes the information stored for open  output  files:
+**  'file' holds the file identifier which is  received  from  'SyFopen'  and
+**  which is passed to  'SyFputs'  and  'SyFclose'  to  identify  this  file.
+**  'line' is a buffer that holds the current output line.
+**  'pos' is the position of the current character on that line.
+*/
+struct TypOutputFile {
+    // pointer to the previously active output
+    struct TypOutputFile * prev;
+
+    BOOL isstream;
+    BOOL isstringstream;
+    Obj  stream;
+    Int  file;
+
+    char line[MAXLENOUTPUTLINE];
+    Int  pos;
+    BOOL format;
+    Int  indent;
+
+    /* each hint is a tripel (position, value, indent) */
```

Comment:
I know it's not your own typo, but we may as well fix it:
```suggestion
    /* each hint is a triple (position, value, indent) */
```

Reply:
Done, plus a few more (except what you already fixed in #4139)

Reply:
Nice! The typos that you fixed also exist in `io.c`, I think, but I'm totally happy for them to be fixed some other time.

Reply:
Also fixed in `io.c`. I hate it when those comments drift apart -- should probably get rid of more of the duplicates. Thanks for catching this one, too